app/components/ActionLinks/index.js

```diff
@@ -95,6 +101,12 @@ const ActionLinks = ({
           </ActionLink>
         )}
 
+        {isLicenceUser && licencesUrl && (
+          <ActionLink url={`${licencesUrl}`} image="/img/ICON_Licences.png" testId="licence-link">
+            HDC and Licences
```

Comment:
should this `HDC and licences`?
(I know not in the real world, but in "consistency land")

Reply:
Probably but Eddie said that was the wording